app/frontend/components/pages/dashboard/dashboardPage.tsx

```diff
@@ -243,22 +244,25 @@ const DashboardPage = () => {
     shouldShowSendTransactionModal,
     shouldShowDelegationModal,
     shouldShowVotingDialog,
-  } = useSelector((state) => ({
-    activeMainTab: state.activeMainTab,
-    displayInfoModal: state.displayInfoModal,
-    isShelleyCompatible: state.isShelleyCompatible,
-    shouldShowNonShelleyCompatibleDialog: state.shouldShowNonShelleyCompatibleDialog,
-    shouldShowPremiumBanner: shouldShowPremiumBannerSelector(state),
-    shouldShowSaturatedBanner: state.shouldShowSaturatedBanner,
-    activeAccountIndex: state.activeAccountIndex,
-    shouldShowExportOption: shouldShowExportOptionSelector(state),
-    shouldShowWantedAddressesModal: state.shouldShowWantedAddressesModal,
-    shouldShowConfirmTransactionDialog: state.shouldShowConfirmTransactionDialog,
-    shouldShowSendTransactionModal: state.shouldShowSendTransactionModal,
-    shouldShowDelegationModal: state.shouldShowDelegationModal,
-    shouldShowVotingDialog: state.shouldShowVotingDialog,
-    walletOperationStatusType: state.walletOperationStatusType,
-  }))
+  } = useSelector(
+    (state) => ({
+      activeMainTab: state.activeMainTab,
+      displayInfoModal: state.displayInfoModal,
+      isShelleyCompatible: state.isShelleyCompatible,
+      shouldShowNonShelleyCompatibleDialog: state.shouldShowNonShelleyCompatibleDialog,
+      shouldShowPremiumBanner: shouldShowPremiumBannerSelector(state),
+      shouldShowSaturatedBanner: state.shouldShowSaturatedBanner,
+      activeAccountIndex: state.activeAccountIndex,
+      shouldShowExportOption: shouldShowExportOptionSelector(state),
+      shouldShowWantedAddressesModal: state.shouldShowWantedAddressesModal,
+      shouldShowConfirmTransactionDialog: state.shouldShowConfirmTransactionDialog,
+      shouldShowSendTransactionModal: state.shouldShowSendTransactionModal,
+      shouldShowDelegationModal: state.shouldShowDelegationModal,
+      shouldShowVotingDialog: state.shouldShowVotingDialog,
+      walletOperationStatusType: state.walletOperationStatusType,
+    }),
+    isEqual
```

Comment:
I'd put here a comment explaining why is this needed and if you found the solution somewhere, I'd reference the resource